manager/dispatcher/dispatcher.go

```diff
@@ -12,6 +12,7 @@ import (
 	"google.golang.org/grpc/transport"
 
 	"github.com/docker/go-events"
+	metrics "github.com/docker/go-metrics"
```

Comment:
nit: providing a separate name isn't required. `go-metrics` can be used as `metrics` in this file.

Reply:
Totes. but I saw it done another place in swarmkit, and thought 'metrics.something' suited the style better than 'go-metrics.something'

Reply:
It will still be `metrics.something`.

Reply:
Sorry, yea I see what you mean. Whats the way to figure out the package name without looking at the code ? Also, any reason why its not the same as the directory name ? @nishanttotla

Reply:
I don't think there is a way without looking at the code, but by convention, packages named `go-foo` typically have the package name set to `foo`. I don't think hyphens are allowed in package names.